ARDroneLib/VP_SDK/VP_Os/linux/intrin.h

```diff
@@ -9,22 +9,37 @@
 
 #if TARGET_CPU_X86 == 1
 
-static INLINE uint32_t _BitScanReverse(uint32_t* index, uint32_t mask)
+/*static INLINE uint32_t _BitScanReverse(uint32_t* index, uint32_t mask)
```

Comment:
I think after fixing the above mentioned typo, you should be able to un-comment these lines back to their original version and compile this on ARM. Please let me know how it goes.